source/MapPanel.cpp

```diff
@@ -1471,6 +1479,8 @@ void MapPanel::DrawMissions()
 			DrawPointer(waypoint, missionCount[waypoint].drawn, waypointColor);
 		for(const Planet *stopover : mission.Stopovers())
 			DrawPointer(stopover->GetSystem(), missionCount[stopover->GetSystem()].drawn, waypointColor);
+		for(const System *ping : mission.Pings())
+			DrawPointer(ping, missionCount[ping].drawn, waypointColor);
```

Comment:
When you both set a system as a waypoint and ping it, you get two dark blue markers from the same mission on a single system. It doesn't make much sense to me, unless we're going to make pings and waypoints look different from each other.

Reply:
I'd question why a mission would both mark/ping a system and use it as a waypoint in the first place. Even if there would be a good reason to waypoint and mark a system, I believe a system that is a waypoint and contains a stopover will also show two markers of the same type; should that also change, or how would that case be different where it shouldn't change?

Reply:
Hmm, that's a good point; it can be changed in a separate PR if we want.